registry/obo_context.jsonld

```diff
@@ -20,14 +22,20 @@
         "CEPH": "http://purl.obolibrary.org/obo/CEPH_",
         "CHEBI": "http://purl.obolibrary.org/obo/CHEBI_",
         "CHEMINF": "http://purl.obolibrary.org/obo/CHEMINF_",
+        "CHIRO": "http://purl.obolibrary.org/obo/CHIRO_",
```

Comment:
there are no terms in CHIRO that could be resolved by this prefix. Should there still be an entry for this? I'm curating which entries in OBO Foundry don't have any own terms in Bioregistry, so you could leverage that

Reply:
This context is autogenerated by the OBO foundry from its ontologies - even if we don't have any terms right now, we would not know whether there are not any in the future, so we reserve the namespace..

Reply:
Some older discussion here https://github.com/OBOFoundry/OBOFoundry.github.io/issues/872

Ideally there would be a way in OBO to denote an ontology that mint no IDs. Really CHIRO should go away as soon as we figure a way to overlay onto CHEBI

Reply:
There's a flag in the bioregistry for ontologies that don't mind IDs ;)